package.json

```diff
@@ -5,10 +5,11 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "db:build": "npm run build && node dist/db/seed/index.js --resolveJsonModule",
     "build:dev": "tsc --watch --preserveWatchOutput",
     "start:dev": "nodemon dist/index.js & webpack -w",
     "dev": "concurrently 'npm run build:dev' 'npm run start:dev'",
-    "build": "tsc",
+    "build": "tsc --resolveJsonModule",
```

Comment:
Can you move this into the ts config? It should really live there so that all the TS settings are in one place